lib/app.js

```diff
@@ -61,6 +62,13 @@ app.post('/cloud/:datasetId', function(req, res) {
 var excludedPaths = ['/authpolicy', '/file'];
 app.use(require('fh-wfm-user/lib/middleware/validateSession')(mediator, mbaasApi,excludedPaths));
 
+
+//Initialising the fh-wfm-sync data sets for each module
+wfmSync.init(mediator, mbaasApi, 'workorders', config.get("syncOptions"));
+wfmSync.init(mediator, mbaasApi, 'result', config.get("syncOptions"));
+wfmSync.init(mediator, mbaasApi, 'workflows', config.get("syncOptions"));
+wfmSync.init(mediator, mbaasApi, 'messages', config.get("syncOptions"));
```

Comment:
I would see it that each module would have wrapper method where we would pass sync options. This would be more framework way as cloud apps would not need to know what is actual dataset name supported by module.